### ISSUE TITLE
Fanstastic app, just needs little tweeks to be perfect

### ISSUE THREAD

**Issue by Thejoker5212** [RadAd/RadSwitch#2]
Hey, i've been using you're app for like 5 months now, it's a very unique simple app but it does the job perfectly, it seperates the apps for each monitors, and it's quick, one thing i hated, is the GUI, it's a bit confusing, when i switch to an app, the order in the list of the GUI get reversed, ik it's a normal thing, but could make the apps stable in the list even when i switch , basically can you make taskbar like switching,

**Comment by RadAd** [comment]
The order is the same as the default alt+tab interface it uses the window z-order which is effectively a last recently used order.

**Comment by Thejoker5212** [comment]
Totally agree, man, it's indeed the same order as the default alt+tab

not to whine or anything, this is some things i come across in the app, that would like to be changed, in case you had the time to do an update: 

1**. Can you remove the app you're currently using in the list when you hit alt+tab, and show only the others apps that are open**

for example i was using M.edge and i want to switch to firefox this is the list that it will be shown in the app:
![Capture d'écran 2024-06-27 203712](https://github.com/RadAd/RadSwitch/assets/134554952/72b8bd39-1c92-4833-be4b-e434361fe0a5)
with the update:
![Capture d'écran 2024-06-27 204146](https://github.com/RadAd/RadSwitch/assets/134554952/642b6c87-63c6-47ad-8a8b-0f727ff5080b)
it will show only the other app

2. Can you do an option where you change the GUI  (Macos style)
![image](https://github.com/RadAd/RadSwitch/assets/134554952/ded833fb-73d9-4923-97c4-0d33214a01ce)

Basically, i just want a "centred taskbar" where it shows only the new opened  apps without reversing orders when i switch apps.

i found you're app the only workable alt+tab alternative bcs i have multiple monitors, and as you know alt+tab of windows 11 suck ass, so that's why i reached you, i recognize you're talent, hope you see this message mate.
